Dockerfile.spark

```diff
@@ -12,7 +12,11 @@ RUN apt-get update && apt-get install -yq --no-install-recommends --force-yes \
     rm -rf /var/lib/apt/lists/*
 
 RUN easy_install3 pip py4j
-RUN pip install --upgrade setuptools
+RUN pip install --upgrade setuptools && \
```

Comment:
You are running this twice. Remove the first `pip install --upgrade setuptools`

Reply:
This is what I was referring to in my comment in previous thread. Easy_install3 install pip for python 3.5 and I am trying to install pip for the python 2.7 that Livy uses to build clientAPI. It is fine to remove the pip in line 14 and line 15 entirely but it will have downstream effects on line 51, 52 and general bash usage.

Hope this help demonstrate my intent:
```
root@spark:/# pip --version
pip 20.0.2 from /usr/local/lib/python3.5/dist-packages/pip-20.0.2-py3.5.egg/pip (python 3.5)**
root@spark:/# python -m pip --version
pip 20.0.2 from /root/.local/lib/python2.7/site-packages/pip (python 2.7)
```

If I remove the line, `pip` will either not be available using my current installation method or will be bound to Python 2.7 if I use easy-install
```
root@1f38eb8e5a67:/# pip --version
pip 20.0.2 from /usr/local/lib/python2.7/dist-packages/pip-20.0.2-py2.7.egg/pip (python 2.7)
```

Reply:
I am totally open to rebind the pip variable to 2.7 but just want to confirm this before I made the change

Reply:
Understood! Thanks for the detailed explanation